Doc/library/concurrent.futures.rst

```diff
@@ -142,6 +145,12 @@ And::
       control the threading.Thread names for worker threads created by
       the pool for easier debugging.
 
+   .. versionadded:: 3.7
```

Comment:
For a new function parameter, this should be ``versionchanged`` (``versionadded`` means the function itself is new). Unfortunately, it seems we did the same mistake above for ``thread_name_prefix``...